api/src/tests/resource_groups.rs

```diff
@@ -49,8 +49,8 @@ async fn test_gen_resource_group() {
     context.publish_package(&mut admin1, txn).await;
 
     // Read default data
-    let primary = format!("0x{}::{}::{}", admin0.address(), "primary", "Primary");
```

Comment:
why are we removing "0x" here? Should we do a to_hex() here as well?

Reply:
In this case we're just using it for comparison in the tests and there are no goldens related to it or anything so it's fine. I could do `to_hex` but it wouldn't really be any different.

Reply:
Ahh I didn't realize these were tests. sgtm